.github/workflows/dco-check.yml

```diff
@@ -11,69 +11,36 @@
 
 name: PR Check DCO
 on:
-  pull_request:
-    types: [ opened, reopened, synchronize ]
```

Comment:
This is actually [the default](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request) and can be omitted.